app/src/main/java/com/codesoom/assignment/handlers/TaskHttpHandler.java

```diff
@@ -0,0 +1,287 @@
+package com.codesoom.assignment.handlers;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import static com.codesoom.assignment.App.*;
+
+/**
+ * Task 도메인 리퀘스트를 처리합니다.
+ */
+
+public class TaskHttpHandler implements HttpHandler {
+    private List<Task> tasks = new ArrayList<Task>();
+    private final ObjectMapper objectMapper = new ObjectMapper();
+    private int responseCode;
+
+    /**
+     * 사용자가 Request 를 보내면 실행되는 메서드 입니다.
```

Comment:
메소드에 대해 설명할 때 `~메소드입니다` 할 필요는 없습니다.
```suggestion
     * Http 요청을 처리합니다.
```